code/index.html

```diff
@@ -1,30 +1,98 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
+    <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Comfortaa:700|Lobster&display=swap" rel="stylesheet">
+    <title>Project 3: The business Site</title>
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header>
+      <div class="big-text-container">
+        <div>
+          <h1>SEEING THE FOREST FOR MORE TREES</h1>
+          <h4>Contemporary forestry in Sweden is largely based on monocultures, usually pine or spruce - we give you the knowledge.</h4>
+        </div>
+      </div>
+    </header>
 
-    <!-- Signup form -->
+    <div class="box-container">
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+          <!-- MY MAGIC SIGN-UP FORM -->
+      <section class="my-form">
+        <div>
+          <form action="http://httpbin.org/anything" target="_blank" method="POST">
+            <h2>LET'S STAY IN TOUCH - IT'S ALL ABOUT TREES:</h2>
```

Comment:
A tip is to write this in normal letters and instead in css make it uppercase. Then it will look better in other pages, like in Google, but it will still look like you want styling wise!

Reply:
Thank you Erica! You are so right about the letters, I missed on that. 🌼